README.md

```diff
@@ -53,17 +53,12 @@ If you plan on extending the features of MSync 2, please copy and use the sample
 
 You need to install some things manually on the Server in order for MSync to work. 
 
-First you need libmysql on windows or libmysqlclient on linux. You can find them here:
-Windows: https://github.com/FredyH/MySQLOO/raw/master/MySQL/lib/windows/libmysql.dll
-Linux: https://github.com/FredyH/MySQLOO/raw/master/MySQL/lib/linux/libmysqlclient.so.18
-Put the file in the same folder where your scrds.exe or your scrds_linux file is.
-
-Then you need MySQLoo 9 or higher. You can download MySQLoo from here:
-https://github.com/FredyH/MySQLOO/releases
-Select win32 if your server runs on windows, otherwise use the linux dll.
-Put the dll file in your lua/bin folder. If you don't have one, create it.
-it should look like this:
+First you need gmsv_mysqloo_win32.dll on windows or gmsv_mysqloo_linux.dll on linux. You can find them here:
+Windows: [https://github.com/FredyH/MySQLOO/raw/master/MySQL/lib/windows/libmysql.dll](https://github.com/FredyH/MySQLOO/releases)
+Linux: [https://github.com/FredyH/MySQLOO/raw/master/MySQL/lib/linux/libmysqlclient.so.18](https://github.com/FredyH/MySQLOO/releases)
```

Comment:
These links are not correct. Since both files can be found there I don't see a reason to link the OS specific files explicitly. Just link to the releases directly